clients/web/src/lib/load-fathom.ts

```diff
@@ -0,0 +1,8 @@
+import { createScriptLoader } from "@solid-primitives/script-loader";
+
+export function loadFathom() {
+  const fathomUrl = import.meta.env.VITE_FATHOM_URL;
+  const fathomId = import.meta.env.VITE_FATHOM_ID;
+  if (fathomUrl && fathomId)
+    createScriptLoader({ src: fathomUrl, "data-site": fathomId, defer: true });
```

Comment:
We can only test in production if there is some CSP headers blocking the script. Most ad blockers will block it. too.